### ISSUE TITLE
Production usage

### ISSUE THREAD

**Issue by sberryman** [disintegrator/seneca-amqp-transport#2]
Are you using this in any of your projects right now? I'm just getting started on Seneca and we use RabbitMQ heavily in production right now. I would love to use this vs standing up and maintaining Kafka and wondering if you have an suggestions or recommendations on setup.

**Comment by disintegrator** [comment]
Hi @sberryman,
I intended to use this lib in production but didn't get a chance to at the time and now most of my work is in Go (I continue maintain this lib). The only way I can address your concerns is to invite you to look at the source code of this library as it's quite small and see if you agree with the implementation. It's just a wrapper around amqlib. I'd say having tests might boost confidence in using the lib but I don't have any time planned for writing them soon (contributions welcome :)).

As for RabbitMQ vs Kafka vs whatever: The only recommendation I can give is to worry more about understanding the features and failure modes for the tools you pick ([RabbitMQ][1] or [Kafka][2]) and to put in adequate monitoring and failure mitigation/recovery for the services you use (dropped messages, network partitions, etc...).

I hope I've answered your question. I'll leave it to you to close this issue unless you have more questions

[1]: https://aphyr.com/posts/315-call-me-maybe-rabbitmq
[2]: https://aphyr.com/posts/293-call-me-maybe-kafka
